app/src/main/java/org/mozilla/fenix/experiments/NimbusSetup.kt

```diff
@@ -83,10 +85,7 @@ fun createNimbus(context: Context, url: String?): NimbusApi =
     } catch (e: Throwable) {
         // Something went wrong. We'd like not to, but stability of the app is more important than
         // failing fast here.
-        if (isSentryEnabled()) {
-            Sentry.capture(e)
```

Comment:
Ugh: https://github.com/mozilla-mobile/fenix/issues/16330 -> reopened to review remaining call sites, luckily very few.